pkg/insights/insightsclient/insightsclient.go

```diff
@@ -184,6 +184,8 @@ func (c *Client) Send(ctx context.Context, endpoint string, source Source) error
 	switch resp.StatusCode {
 	case http.StatusOK:
 		counterRequestSend.WithLabelValues(c.metricsName, "200").Inc()
+	case http.StatusCreated:
+		counterRequestSend.WithLabelValues(c.metricsName, "201").Inc()
 	case http.StatusAccepted:
 		counterRequestSend.WithLabelValues(c.metricsName, "202").Inc()
```

Comment:
Do we need to exhaustively list all possible 2xx codes here?  We know they are [all successful][1].

[1]: https://tools.ietf.org/html/rfc7231#section-6.3

Reply:
No we sure don't.  I refactored the section a bit.